packages/app/src/cli/commands/app/logs.ts

```diff
@@ -21,8 +21,22 @@ export default class Logs extends Command {
 
   static description = this.descriptionWithoutMarkdown()
 
-  static flags = {
-    ...Dev.flags,
+  static flags: any = {
+    ...Object.fromEntries(
+      Object.entries(Dev.flags).filter(
+        ([k]) =>
+          ![
+            'theme',
+            'checkout-cart-url',
+            'no-update',
+            'notify',
+            'skip-dependencies-installation',
+            'subscription-product-url',
+            'theme-app-extension-port',
+            'tunnel-url',
+          ].includes(k),
+      ),
```

Comment:
I'd have like to use variables instead of this giant list inside of a hash but doing so also adds those variables to the manifest file which isn't something I want.